CMS/urls.py

```diff
@@ -18,6 +18,10 @@
 from django.conf import settings
 from django.conf.urls.static import static
 
+admin.site.site_header = "SpaceYaTech CMS Admin"
```

Comment:
What is the difference between  "SpaceYaTech CMS Admin" and  "SpaceYaTech Admin Portal"  if none then we could merge them and give it a  simple name like "SpceYaTech Portal" without using the word Admin

Reply:
these titles are for the admin site so thats why I put the 'Admin' in there just to indicate that we are currently on admin